### ISSUE TITLE
ERROR: unable to select packages libcrypto.so.3 required by: libsrt-1.5.1-r0

### ISSUE THREAD

**Issue by FredStober** [angristan/docker-pleroma#68]
I got the following error message during the build process on an ARM64 machine:

```
ERROR: unable to select packages:
  so:libcrypto.so.3 (no such package):
    required by: libsrt-1.5.1-r0[so:libcrypto.so.3]
```
I fixed it by adding
`echo "http://nl.alpinelinux.org/alpine/latest-stable/main"
`
into the command 
`RUN echo "http://nl.alpinelinux.org/alpine/latest-stable/community" >> /etc/apk/repositories     && apk update     && apk add git gcc g++ musl-dev make cmake file-dev     exiftool imagemagick libmagic ncurses postgresql-client ffmpeg
`

**Comment by sanderdatema** [comment]
Yep, that seems to fix it. Would you like to do the honors and make a PR? I can do it, but since you found it first... :)
